### ISSUE TITLE
Electret mic questions

### ISSUE THREAD

**Issue by jatahan** [Audio-Injector/stereo-and-zero#52]
Hi there,

Following up from an earlier email, can anyone tell me what the bias voltage for the electret mic is?

And I think the unit ships with an electret capsule, what's the part number?

Thanks,
Jem

**Comment by Srivallabha-k** [comment]
Hello, 

Does anyone know the model number of the Electret microphone which is supplied with Zero soundcard?

Thanks
KS

**Comment by flatmax** [comment]
Similar specs to the WM-52B electret microphone.

**Comment by Srivallabha-k** [comment]
Thanks for your response, Matt. But, can you point me to the exact microphone which is in use, that would really help me?

**Comment by flatmax** [comment]
The model is 52D electret microphone.

it is similar to the wm-52b microphone

**Comment by Srivallabha-k** [comment]
Hello Matt,

Really appreciate your response. I am facing an issue with Zero soundcard, randomly we get to hear blips/pops in the recorded audio files. We noticed this happening in two different samples, Please listen to attached audio sample at around 2 seconds you will hear a pop. It is more frequent in one hardware over another. Is this a known issue? How do we fix this?  If it helps i am recording audio at 44.1khz, S16_LE format, wav format with rest of parameters being the default.  Your response would be highly appreciated. 

Thank you.

[2020_03_11-15_40_16.zip](https://github.com/Audio-Injector/stereo-and-zero/files/4321410/2020_03_11-15_40_16.zip)

**Comment by flatmax** [comment]
Are the pops to do with overruns ? When ALSA misses capturing samples 
because the buffer is full ?
Do you see anything reported about xruns ?

**Comment by Srivallabha-k** [comment]
I don't think it is something to do with buffer. I tried varying buffer size and that didn't have any effect. Also, if it has to do with buffers the frequency of occurrence will be precise which isn't the case.  As reported in #53, it is not dependent on buffer size. As per our tests on two different setups, It also slightly varies from one hardware to another in terms of frequency of occurrence,

**Comment by jatahan** [comment]
I also have the pops on recording and my case it's interference from the wifi. When I have the AI board mounted directly over an Zero W (i.e. gpio pins aligned, 10mm space between boards) I get pops on recording that align with wifi transmission. To test, try recording while pinging the pi at 1s intervals with no other traffic on the interface and you should get pops separated by 1s on your recording.  I haven't got around to testing with different alignments yet.

**Comment by flatmax** [comment]
Oh yep - a good reason to isolate the sound card. There is likely currents being induced by EMI, which  sullies the ground plane.
